ipmi-config/ipmi-config.h

```diff
@@ -276,6 +276,14 @@ struct ipmi_config_state_data
   int enable_user_after_password_len;
   ipmi_config_enable_user_after_password_t *enable_user_after_password;
 
+  /* lamont: caching to make ipv6 go faster */
+  int ipv6_ipv4_support_initialized;
+  uint8_t ipv6_ipv4_support_channel_number;
+  uint8_t ipv6_ipv4_support_supports_ipv6;
```

Comment:
This was more opportunistically added for future use, than for need.  Would you prefer that it linger for later use as we actually check that ipv6 is supported before trying to set ipv6 addresses, or shall we drop it until we actually have something that uses it?